Project.toml

```diff
@@ -32,6 +32,7 @@ Polynomials = "f27b6e38-b328-58d1-80ce-0feddd5e7a45"
 Primes = "27ebfcd6-29c5-5fa9-bf4b-fb8fc14df3ae"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
+PseudoPotentialIO = "cb339c56-07fa-4cb2-923a-142469552264"
```

Comment:
Version bounds for `PseudoPotentialIO`

Reply:
I'll wait for the registration to go through; I'll bump the version with some more recent changes and then add the bounds.